### PR TITLE
Add ability to view previous edits of a status in admin UI

### PR DESCRIPTION
**Pull request by Gargron**
Screen Shot 2022-10-26 at 01 37 51

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

File: app/javascript/mastodon/components/status_action_bar.js [hunk 1]
```diff
@@ -280,7 +280,7 @@ class StatusActionBar extends ImmutablePureComponent {
     }
 
     if (writtenByMe) {
-      // menu.push({ text: intl.formatMessage(messages.edit), action: this.handleEditClick });
+      menu.push({ text: intl.formatMessage(messages.edit), action: this.handleEditClick });
```
Comment (ClearlyClaire) [inline app/javascript/mastodon/components/status_action_bar.js:283]: This should be in another PR. Preferably with support for editing media attachments.

File: app/policies/status_policy.rb [hunk 2]
```diff
@@ -12,9 +12,11 @@ def index?
...
 
   def show?
-    return false if author.suspended?
-
-    if requires_mention?
+    if role.can?(:manage_reports) && record.reported?
+      true
+    elsif author.suspended?
+      false
+    elsif requires_mention?
```
Comment (ClearlyClaire) [inline app/policies/status_policy.rb:19]: I'm afraid this could cause moderators to see posts from suspended authors in the non-admin interface, causing confusion. Maybe this should be a separate policy.

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

**Review by ClearlyClaire — APPROVED** [review]

**Comment by joenepraat** [comment]
Why is 'reblogs' used? That is not in sync with 'boosts'.
